nondex-maven-plugin/src/main/java/edu/illinois/nondex/plugin/NonDexMojo.java

```diff
@@ -69,7 +69,7 @@ public void execute() throws MojoExecutionException, MojoFailureException {
                     this.mavenSession, this.pluginManager,
                     Paths.get(this.baseDir.getAbsolutePath(), ConfigurationDefaults.DEFAULT_NONDEX_DIR).toString());
 
-        allExceptions = this.executeSurefireExecution(allExceptions, cleanExec);
+        this.executeSurefireExecution(allExceptions, cleanExec);
```

Comment:
Add comment here explaining why we are ignoring the returned exception (if any exists).